### ISSUE TITLE
[Question] Support for Staticman comments

### ISSUE THREAD

**Issue by VincentTam** [lingxz/er#8]
Hi, this is the author of Beautiful Hugo's Staticman nested comment support halogenica/beautifulhugo#222.

Anyone using this theme would be interested in having Staticman?

**Comment by rootkea** [comment]
>  I'm completely open to having comments support in a separate branch/PR for people to switch to it if they wish to.  

Well, this could work too. Thanks for being open to suggestions. :)

**Comment by lingxz** [comment]
yes, I'm just saying I won't merge it, but we can leave it in a separate branch. There's no need for a dev branch for now, you can just raise PR against master.

**Comment by VincentTam** [comment]
I'll try two weeks later.
